app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -68,6 +68,26 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
+  let monitoring_url;
+  if (experimentData.monitoring_dashboard_url) {
+    monitoring_url = (
+      <>
+        <h3 className="my-4">Monitoring</h3>
+        <p>
+          The live monitoring dashboard can be found here:
+          <a
+            href={experimentData.monitoring_dashboard_url}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            {" "}
```

Comment:
nit: do we want the spacer _outside_ of the linked text? Like, maybe at end of L77 above?

Reply:
Eslint fix adds this automatically, if I take it out it'll just put it back.